### PR TITLE
Add manifest file for WebRTC crashtest job

### PR DESCRIPTION
**Pull request by dminor**
This is based on the existing crashtest config, I just dropped the chunks to one and renamed the job.

### CODE REVIEW

File: configs/crashtests-webrtc.ini [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+# test_name is a descriptor used by runtestsremote.py to
+# determine which of the downloaded unit tests to run.
+#
+# test_manifest is the path to the location of the test_manifest.
+#
+# test_package_names is a list of the keys for test_packages.json
+# which will need to be downloaded in order to run the test.
+
+test_name = crashtest
```
Comment (bclary) [inline configs/crashtests-webrtc.ini]: This needs to be distinct from the other tests. crashtest-webrtc should do it. The full formal name will end up being autophone-crashhtest-webrtc when we want to put this in trychooser.

File: configs/crashtests-webrtc.ini [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+test_manifest = reftest/tests/testing/crashtest/autophone-webrtc.list
```
Comment (bclary) [inline configs/crashtests-webrtc.ini]: I know it is redundant with the parent directory but to follow the adjacent file, lets call it autophone-crashtests-webrtc.list

File: configs/crashtests-webrtc.ini [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+unittest_defaults = configs/unittest-defaults.ini
+
+# How many times to run the tests per phone.
+iterations = 1
+
+# How many chunks for the test
+total_chunks = 1
+
+[treeherder]
+job_name = Autophone Crashtest
```
Comment (bclary) [inline configs/crashtests-webrtc.ini]: Autophone WebRTC Crashtest

File: configs/crashtests-webrtc.ini [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+job_symbol = Cw
```
Comment (bclary) [inline configs/crashtests-webrtc.ini]: Is Cwrtc too long? Maybe Cwrt ? Just want to make sure we don't have a better candidate to use Cw as the symbol.

**Comment by dminor** [comment]
Updated and pushed to: https://github.com/mozilla/autophone/commit/9dd3706df1c4904e6ea0f27c07ea08142b6eae9f

**Pull request closed without merging**